src/com/palmergames/bukkit/towny/command/InviteCommand.java

```diff
@@ -309,15 +309,15 @@ public static void sendInviteList(Player player, List<Invite> list, int page, bo
 				}
 			} else { // So it's not from the sender, then it's from the receiver so
 				if (invite.getReceiver() instanceof Resident) {
-					output = Colors.Blue + ((Town) invite.getSender()).getName() + Colors.Gray + " - " + Colors.Green + name;
+					output = Colors.Blue + invite.getSender().getName() + Colors.Gray + " - " + Colors.Green + name;
 					object = TownySettings.getLangString("town_sing");
 				}
 				if (invite.getReceiver() instanceof Town) {
-					output = Colors.Blue + ((Nation) invite.getSender()).getName() + Colors.Gray + " - " + Colors.Green + name;
+					output = Colors.Blue + invite.getSender().getName() + Colors.Gray + " - " + Colors.Green + name;
 					object = TownySettings.getLangString("nation_sing");
 				}
 				if (invite.getReceiver() instanceof Nation) {
-					output = Colors.Blue + ((Nation) invite.getSender()).getName() + Colors.Gray + " - " + Colors.Green + name;
+					output = Colors.Blue + invite.getSender().getName() + Colors.Gray + " - " + Colors.Green + name;
 					object = TownySettings.getLangString("nation_sing");
 				}
```

Comment:
Should probably be this if the casting isn't needed to work. Have you tested this?

```
			} else { // So it's not from the sender, then it's from the receiver so
				output = Colors.Blue + invite.getSender().getName() + Colors.Gray + " - " + Colors.Green + name;
				if (invite.getReceiver() instanceof Resident) {
					object = TownySettings.getLangString("town_sing");
				}
				if (invite.getReceiver() instanceof Town || invite.getReceiver() instanceof Nation) {
					object = TownySettings.getLangString("nation_sing");
				}
			}
```